content/updates/2017-06-12-release-notes.md

```diff
@@ -0,0 +1,25 @@
++++
+date = "2017-06-12"
+title = "Platform Release Notes: June 12, 2017"
++++
+
+Curious what’s new that you might find helpful as a cloud.gov application developer? Here are highlights from our platform updates over the past week and a half.
+<!--more-->
+
+### Fixed
+
+- We finished migrating all tenants from East/West into GovCloud! Thank you to all our customers for their patience and cooperation in making this move successful.
```

Comment:
Our [most recent set of platform release notes](https://cloud.gov/updates/2017-06-01-release-notes/) said "cloud.gov documentation no longer provides instructions for using the deprecated East/West environment, because all customer applications have migrated to the GovCloud environment." so this seems redundant - do we need to mention it again?